testing/trino-server-dev/etc/catalog/scylladb.properties

```diff
@@ -0,0 +1,6 @@
+connector.name=scylladb
```

Comment:
There is no need to add this property since we have a query runner.

Reply:
Should i also remove the same property from cassandra catalog than?

Reply:
No need to remove the existing catalog properties.